.circleci/config.yml

```diff
@@ -6,10 +6,10 @@ orbs:
 parameters:
```

Comment:
Should we introduce a develop approach? or wait until later

Reply:
Yes need to introduce the develop approach. But maybe later, once we get 1 release into Live?